### PR TITLE
Rupyainr - Indian currency symbol for website

### PR DESCRIPTION
**Pull request by VinuRaj**
Indian currency symbol (Rupee) in web font along with easy Javascript implementation for websites.

Have updated all files with valid website details that tell about this project.

Please verify

### CODE REVIEW

File: files/rupyainr/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "rupyaINR"
+github = ""
+homepage = "http://www.diagramz.com/"
```
Comment (jimaek) [inline files/rupyainr/info.ini]: The website is not correct again

Comment (VinuRaj) [inline files/rupyainr/info.ini]: missed it, have changed that to correct project website now.
Have committed the changes now

File: files/rupyainr/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author = "rupyaINR"
+github = ""
+homepage = "http://www.rupyaINR.com/"
+description = "India currency web symbol for online websites and apps"
+mainfile = "rupyaINR.js"
```
Comment (jimaek) [inline files/rupyainr/info.ini]: Its better to use `rupyaINR.min.js` here

**Comment by jimaek** [comment]
Once you do all the changes please squash your commits into 1

File: files/rupyainr/info.ini (hunk 2 above)
Comment (VinuRaj) [inline files/rupyainr/info.ini]: This has been done now.

**Comment by megawac** [comment]
```
git reset --soft HEAD~9
git commit -am "Add Rupyainr"
git push -f origin master
```

**Comment by VinuRaj** [comment]
Sorry for the late reply, it has been done now..

**Pull request merged**